tests/schemas/fts_setup_wip.edgeql

```diff
@@ -0,0 +1,522 @@
+start migration to {
```

Comment:
Is this meant to be committed? If so, change the name from `_wip`.

Reply:
@vpetrovykh ?